File: Problem_1/problem_1.erl
-module(problem_1).
-compile(export_all).

start()->
    lists:sum(lists:filter(fun(X) ->
				   (X rem 5 == 0) or (X rem 3 == 0) 
			   end,
			   lists:seq(1, 999))).
			   

File: Problem_1/problem_1.js
function divided(a, b) {
    return a % b == 0;
}

function problem_1() {
    var sum = 0;

    for (var i = 1; i < 1000; i++) {
	if(divided(i, 3) || divided(i, 5))
	    sum += i;
    }

    return sum;
}

document.write('<b>result: </b>' + problem_1());